bin/src/daemon.rs

```diff
@@ -142,12 +142,17 @@ fn run_ctrl_c_thread(
 
 fn update_ui(task_tracker: &TaskTracker,  config: &config::Config, mut ui_shutdown_rx: oneshot::Receiver<()>, mut ui_update_rx: Receiver<framebuffer::DisplayState>) -> JoinHandle<()> {
     static IMAGE_DIR: Dir<'_> = include_dir!("$CARGO_MANIFEST_DIR/static/images/");
+    let mut display_color: framebuffer::Color565;
     let display_level = config.ui_level;
     if display_level == 0 {
         info!("Invisible mode, not spawning UI.");
     }
 
-    let mut display_color = framebuffer::Color565::Green;
```

Comment:
this'll only update the starting color -- colors based on the warning state are determined by the `From` impl of `framebuffer::DisplayState`

Reply:
Yes but we want to keep red for the warning state. I guess this will be a problem if we stop and then start the server though.

Reply:
no like, the implementation of https://github.com/EFForg/rayhunter/blob/main/bin/src/framebuffer.rs#L32-L40 still uses a red/green colorscheme. so if the user pauses the recording and then resumes again, it'll go back to green instead of blue if colorblind mode is enabled.

Reply:
yea I understood that. Uploaded a fix.